ui/src/pages/version/components/forms/components/ImageBuilderSection.js

```diff
@@ -1,64 +1,72 @@
 import { FormLabelWithToolTip, useOnChangeHandler } from "@caraml-dev/ui-lib";
-import { 
-    EuiFieldText, 
-    EuiFlexGroup, 
-    EuiFlexItem, 
-    EuiFormRow, 
+import {
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiTitle,
 } from "@elastic/eui";
 
 
 export const ImageBuilderSection = ({
-    imageBuilderResourceConfig,
-    onChangeHandler,
-    errors = {},
-  }) => {
+  imageBuilderResourceConfig,
+  onChangeHandler,
+  errors = {},
+}) => {
+  const {onChange} = useOnChangeHandler(onChangeHandler);
 
-    const { onChange } = useOnChangeHandler(onChangeHandler);
-
-    return (
+  return (
+    <>
+      <EuiTitle size="xs">
+        <h4>{"Image Building Resources"}</h4>
+      </EuiTitle>
     <EuiFlexGroup direction="row">
-        <EuiFlexItem>
+      <EuiFlexItem>
         <EuiFormRow
-            label={
+          label={
             <FormLabelWithToolTip
-                label="Image Builder CPU *"
-                content="To set a higher CPU above platform default for Image Building"
+              label="Image Builder CPU"
+              content="To set a higher CPU above platform default for Image Building"
             />
-            }
-            isInvalid={!!errors.cpu_request}
-            error={errors.cpu_request}
-            fullWidth
+          }
+          isInvalid={!!errors.cpu_request}
+          error={errors.cpu_request}
+          fullWidth
         >
-            <EuiFieldText
+          <EuiFieldText
             placeholder="500m"
             value={imageBuilderResourceConfig?.cpu_request}
             onChange={(e) => onChange("cpu_request")(e.target.value)}
             isInvalid={!!errors.cpu_request}
             name="cpu"
-            />
+            fullWidth
+          />
         </EuiFormRow>
-        </EuiFlexItem>
+      </EuiFlexItem>
 
-        <EuiFlexItem>
+      <EuiFlexItem>
         <EuiFormRow
-            label={
+          label={
             <FormLabelWithToolTip
-                label="Image Builder Memory *"
-                content="To set a higher Mem above platform default for Image Building"
+              label="Image Builder Memory"
```

Comment:
I removed the asterisk `*` because it doesn't seem to suggest anything (this field isn't compulsory or anything).